dask/dataframe/reshape.py

```diff
@@ -106,16 +107,17 @@ def get_dummies(
     """
     if PANDAS_VERSION >= "0.23.0":
         # dtype added to pandas
-        kwargs = {"dtype": dtype}
+        if len(kwargs) == 0:
+            kwargs = {"dtype": dtype}
```

Comment:
Why this change?

Reply:
Because of the reason I had to introduce `kwargs` in `get_dummies` call. I had to do this as currently cudf's get_dummies behavior is slightly different from that of panda's. For the `meta` to be populated for dask to work we would require `cats` attribute to be passed for `cudf.get_dummies` to give the expected columns on an empty dataframe. So instead of creating inconsistency by adding `cats` to dask get_dummies call, moved the kwargs being created below to the function signature itself and having an equivalent check which was being done previously.

Reply:
I see, so you're saying that there is an extra keyword argument that cudf wants, so you want to include `**kwargs` in the dask.dataframe.get_dummies signature?  This makes sense.  

Given that though, I think that you can probably replace these four lines as follows?:

```diff
-if len(kwargs) == 0:
-    kwargs = {'dtype': dtype}
-else:
-    kwargs['dtype'] = dtype
+kwargs['dtype'] = dtype

Reply:
Yes, since `kwargs` would be an empty dict it makes sense to remove the length check. Removed it. 👍